billing/views.py

```diff
@@ -410,7 +410,7 @@ def checkout_session_completed(
 
         self._log_updated([owner])
 
-    def post(self, request: HttpRequest, *args, **kwargs) -> Response:
+    def post(self, request: HttpRequest, *args: Any, **kwargs: Any) -> Response:
```

Comment:
Could we know what the args/kwargs are from stripe?

Reply:
The code doesn't use args/kwargs at all, I think we just add it in as part of the DRF APIView interface.